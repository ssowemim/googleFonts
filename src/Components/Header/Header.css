.Header .App__header {
  top: 25px;
  position: fixed;
  left: 0;
  transition: all .3s ease-out; }

.Header .App__Logo {
  color: #000;
  text-decoration: none;
  width: 160px;
  height: 40px;
  display: flex; }
  .Header .App__Logo span {
    display: inline-block;
    width: 50%;
    padding: 20px 8px;
    line-height: 1px;
    font-size: 22px; }
  .Header .App__Logo .App__Logo--First {
    text-align: right;
    background: #FFDE00;
    font-family: 'Oswald', sans-serif;
    font-weight: 700; }
  .Header .App__Logo .App__Logo--Last {
    text-align: left;
    background: #000;
    color: #fff;
    font-family: 'Marck Script', cursive; }
  .Header .App__Logo:hover .App__Logo--First {
    background: #000;
    color: #fff; }
  .Header .App__Logo:hover .App__Logo--Last {
    background: #FFDE00;
    color: #000; }

.Header .App__SocialIcons {
  margin-top: 30px;
  margin-left: 20px; }
  .Header .App__SocialIcons .App__SocialIcons__Link {
    display: inline-block;
    margin-right: 10px;
    background: #FFDE00;
    font-size: 30px;
    padding: 5px 8px;
    transition: background .3s ease-out;
    color: #000; }
    .Header .App__SocialIcons .App__SocialIcons__Link:hover {
      background: #000;
      color: #fff; }
