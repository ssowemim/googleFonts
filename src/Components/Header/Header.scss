.Header  {
  .App__header {
    top: 25px;
    position: fixed;
    left: 0;
    transition: all .3s ease-out;
  }
  
  .App__Logo {
    color: #000;
    text-decoration: none;
    width: 160px;
    height: 40px;
    display: flex;

    span {
      display: inline-block;
      width: 50%;
      padding:20px 8px;
      line-height: 1px;
      font-size: 22px;
    }

    .App__Logo--First {
      text-align: right;
      background: #FFDE00;
      font-family: 'Oswald', sans-serif;
      font-weight: 700;
    }

    .App__Logo--Last {
      text-align: left; 
      background: #000;
      color: #fff;
      font-family: 'Marck Script', cursive;
    }

    &:hover {
      .App__Logo--First {
        background: #000;
        color: #fff;
      }
      .App__Logo--Last {
        background: #FFDE00;
        color: #000;
      }
    }
  }

  .App__SocialIcons {
    margin-top: 30px;
    margin-left: 20px;

    .App__SocialIcons__Link {
      display: inline-block;
      margin-right: 10px;
      background: #FFDE00;
      font-size: 30px;
      padding: 5px 8px;
      transition: background .3s ease-out;
      color: #000;
      &:hover {
        background: #000;
        color: #fff;
      }
    }
  }
}