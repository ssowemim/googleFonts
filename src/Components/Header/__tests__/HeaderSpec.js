import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow } from 'enzyme';
import Header from './../index';

describe('Header', () => {
    it('renders correctly', () => {
        let tree = renderer.create(
            <Header />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
