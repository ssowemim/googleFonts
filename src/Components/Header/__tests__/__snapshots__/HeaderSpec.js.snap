// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  className="Header"
>
  <header
    className="App__header"
  >
    <a
      className="App__Logo"
      href="/"
    >
      <span
        className="App__Logo--First"
      >
        GFonts
      </span>
      <span
        className="App__Logo--Last"
      >
        Space
      </span>
    </a>
    <div
      className="App__SocialIcons"
    >
      <a
        className="App__SocialIcons__Link"
        href="https://www.linkedin.com/in/pankaj-ladhar-51781137/"
        rel="noopener noreferrer"
        target="_blank"
        title="Pankaj Ladhar linkedin profile "
      >
        <i
          className="fa fa-linkedin"
        />
      </a>
      <a
        className="App__SocialIcons__Link"
        href="https://github.com/pankajladhar/GFontsSpace"
        rel="noopener noreferrer"
        target="_blank"
        title="Pankaj Ladhar github profile "
      >
        <i
          className="fa fa-github"
        />
      </a>
    </div>
  </header>
</div>
`;
